### DISCUSSION TITLE
How can I test the killswitch?

### DISCUSSION THREAD

**Discussion by imthenachoman** [bubuntux/nordlynx#50] [Q&A] [answered]
Is there someway, from inside or outside the container, I can kill the VPN to test the killswitch?

**Comment by ginodesilva** [comment, marked as answer]
There is a way to just stop the container without stopping the entire stack. Other containers will lose their connection (a simple ping to google or a curl ifconfig.me suffices) but a more elegant way is to attach to the nordlynx container and manually shut down the wireguard connection:

`wg-quick down wg0`

After testing from the other container(s), start it again:

`wg-quick up wg0`

**Reply by imthenachoman** [reply]
Great. Thank you!
